app/controllers/identities_controller.rb

```diff
@@ -41,7 +41,7 @@ def authentication_failure
   private
 
   def identity_params
-    params.require(:identity).permit(:first_name, :last_name, :email, :uid, :auth_token, :provider, :location, :description)
+    params.require(:identity).permit(:first_name, :last_name, :email, :uid, :auth_token, :provider, :location, :description, :password_reset_token, :password_reset_sent_at)
```

Comment:
Are `password_reset_token` and `password_reset_sent_at` needed? They aren't submitted as part of the form, are they?